.github/workflows/cli-tests.yaml

```diff
@@ -7,8 +7,6 @@ concurrency:
 
 on:
   pull_request:
-    paths-ignore:
-      - 'docs/**'
```

Comment:
why deleting this?

Reply:
This is part of Mike's quick fix for golangci-lint. Without it, the golangci-lint jobs will not run for docs changes, even though it's still required in order to merge